File: app/src/main/java/br/com/brqtest/database/dao/ClienteDao.java
package br.com.brqtest.database.dao;

import br.com.brqtest.model.Cliente;
import com.j256.ormlite.dao.BaseDaoImpl;
import com.j256.ormlite.support.ConnectionSource;

import java.sql.SQLException;

public class ClienteDao extends BaseDaoImpl<Cliente, Integer> {
    public ClienteDao(ConnectionSource connectionSource) throws SQLException {
        super(Cliente.class);
        setConnectionSource(connectionSource);
        initialize();
    }
}

File: app/src/main/java/br/com/brqtest/databasekotlin/dao/ClienteDao.kt
package br.com.brqtest.databasekotlin.dao

import br.com.brqtest.modelkotlin.Cliente
import com.j256.ormlite.dao.BaseDaoImpl
import com.j256.ormlite.support.ConnectionSource
import java.sql.SQLException


class ClienteDao @Throws(SQLException::class)
constructor(connectionSource: ConnectionSource) : BaseDaoImpl<Cliente, Int>(Cliente::class.java) {
    init {
        setConnectionSource(connectionSource)
        initialize()
    }
}